pubspec.yaml

```diff
@@ -62,9 +62,7 @@ dependencies:
   keyboard_shortcuts: ^0.1.4
   latlong2: ^0.8.1
   matrix:
-    git: 
-      url: git@github.com:linagora/matrix-dart-sdk.git
-      ref: twake-supported
+    path: ../matrix-dart-sdk
```

Comment:
Why do you change it? Can you explain it for team?

Reply:
it's a debug commit, supposed to be in WIP